expan/core/experiment.py

```diff
@@ -141,3 +141,62 @@ def delta(self, method='fixed_horizon', **worker_args):
 
         result['kpis'] = kpis
         return result
+
+    @staticmethod
+    def _quant_apply(value, threshold):
+        if value < threshold:
+            return False
+        else:
+            return True
+
+    def _quantile_filtering(self, df, percentile):
+        flags = None
+        for column in df.columns:
+            if flags is None:
+                flags = df[column].apply(
```

Comment:
can we initialize `flags` with smth like `flags = [False]*len(df[column])` and drop the `if` condition altogether (since the `then` and the `else` arms of the conditional are nearly identical)?